bazel/repository_locations.bzl

```diff
@@ -821,13 +821,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Chrome URL parsing library",
         project_desc = "Chrome URL parsing library",
         project_url = "https://quiche.googlesource.com/googleurl",
-        # Static snapshot of https://quiche.googlesource.com/quiche/+archive/ef0d23689e240e6c8de4c3a5296b209128c87373.tar.gz.
-        version = "ef0d23689e240e6c8de4c3a5296b209128c87373",
-        sha256 = "d769283fed1319bca68bae8bdd47fbc3a7933999329eee850eff1f1ea61ce176",
+        # Static snapshot of https://quiche.googlesource.com/quiche/+archive/578e68d9ae4c25e37bc0d23460f3e2f183fe526a.tar.gz.
+        version = "578e68d9ae4c25e37bc0d23460f3e2f183fe526a",
+        sha256 = "dd13bd6732941a2a869e930503daff44c84eabe92788125bebcbe6b1260587c1",
         urls = ["https://storage.googleapis.com/quiche-envoy-integration/googleurl_{version}.tar.gz"],
```

Comment:
I was wondering with the recent switch of Quiche deps from `quiche.googlesource.com` (https://github.com/envoyproxy/envoy/pull/17732) I was wondering if this dependency can change too? It doesn't look like the `envoy-integration` branch is being updated at the Github location - https://github.com/google/quiche/commits/envoy-integration

Reply:
Yeah I think we can probably move googleurl into Github too. @danzh2010 for more comments.

Reply:
I chatted with Dan, and it seems that @dio has been in charge of this dep.
I currently need to update this dep to do some URL sanitization work. After this PR, @dio can decide how he wants to proceed with the repo location.

Reply:
@danzh2010 @RenjieTang @moderation I think we need to move to GitHub, but that needs Google's decision? Since it will be hosted along with https://github.com/google/quiche (or if it is still only all about envoy, judging from the name: "envoy-integration", probably we may host it on envoyproxy org)? cc. @htuch @yanavlasov. 

PS. Sorry for the very late reply was out for a long time.

Reply:
+1 to move to GH like we did for quiche, but let's discuss on the unrevert PR?